users/models.py

```diff
@@ -116,6 +116,12 @@ def auto_remove_imagefile(sender, instance=None, **kwargs):
     if user and user.image and instance.image != user.image:
         instance.delete_image(user.image)
 
+@receiver(post_save, sender=User)
+def create_auth_token(sender, instance=None, created=False, **kwargs):
+    if created:
+        user = User.objects.filter(id=instance.id).first()
```

Comment:
why? why do you need to query the user using the instance.id if the instance itself is the user?

Also instance is impossible to be `None` as it contains the user object.